kafka/tools/create_volumes.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ux
+
+model=`juju show-model --format=json| jq -r '.| keys[]'`
+machines=`juju status $model --format=json | jq -r '.machines | to_entries[].value."instance-id"'`
```

Comment:
Do you intend to attach 2 volumes to every machine in the model? The title of the PR suggests that those are only necessary for Kafka units.

Reply:
No, but I should run `juju status kafka` instead of using `$model`. It worked for me because my test model is called kafka also.